conf/modules.config

```diff
@@ -1092,3 +1092,17 @@ process {
         ext.args2 = " --min_init_normal_lod 0,5 --min_normal_lod 2,0 --min_init_tumor_lod 1,0 --min_tumor_lod 2,8 --trim_soft_clip "
     }
 }
+
+//
+// PEDDY CHECK
+//
+
+process {
+    withName: '.*PEDDY_CHECK:PEDDY' {
+        ext.args = ''
```

Comment:
Peddy requires the argument `--sites hg38` when running grch38. We should be able to make the switch automatically by using the same logic as for vep
https://github.com/nf-core/raredisease/blob/079c91dacf0781e741cbd56085e3def17cd79e49/conf/modules.config#L715